src/assignment/modals/AddBookmarkFragmentModal.tsx

```diff
@@ -53,15 +58,15 @@ const GET_ADD_BOOKMARK_FRAGMENT_COLUMNS = (): TableColumn[] => [
 		dataType: 'string',
 	},
 	{
-		id: 'contentCreatedAt',
+		id: 'createdAt',
```

Comment:
```suggestion
		id: AddBookmarkFragmentColumn.createdAt,
```

Also applies to other `id`'s